api/GameApi.jsx

```diff
@@ -0,0 +1,134 @@
+import CONSTANTS from "../const";
+
+const buildBotGame = (currentGame) => {
+    const game = {
+        currentPlayer: currentGame.currentPlayer == "red" ? "blue" : "red",
+        pucks: {},
+        fallenPucks: [],
+    };
+    for (const puck of currentGame.pucks) {
+        const position = puck.position.x + ":" + puck.position.y;
```

Comment:
it's more readable to use template literal
```suggestion
        const position = `${puck.position.x}:${puck.position.y}`;
```